.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,34 +1,56 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-
 ---
 
-**Please make sure to read the [FAQ](https://github.com/Alf21/TTT2/wiki/faq) first!**
+## Disclaimer
+
+If you are missing the following information your Issue will be closed!
+
+- "Your Version of TTT2"
+- "Describe the Bug"
+
+Also please make sure to read the [FAQ](https://github.com/TTT-2/TTT2/wiki/faq) first!
+
+## Your Version of TTT2 (mandatory)
+
+Leave one of the following:
+
+- Workshopversion from Steam
+- Latest GitHub Version
+- Non-latest GitHub Version (please provide what you can to specify)
```

Comment:
what do you think of this approach @saibotk @Alf21

Reply:
Thats better in my opinion